lib/tasks/work_index_job.rake

```diff
@@ -5,6 +5,8 @@ task work_index_job: :environment do
 
   Rails.logger.info { "Ran placeholder work_index_job task!" }
 
+  ProcessIndexJobs.new.call
```

Comment:
See above about aligning the PORO and Rake file/task names.  In any event, this rake file/task name needs to be pluralized to end in "Jobs".

Also, note that any changes to the rake file name need to be reflected in the worker launch configuration https://github.com/openstax/unified-deployment/blob/master/cfn/search/worker.yml#L161